checker_test.go

```diff
@@ -57,6 +57,11 @@ type OuterJSON struct {
 
 type boolean bool
 
+func errTestPtr() **errTest {
```

Comment:
Can we define this closer to errTest? Basically move it at the end of the file?

Reply:
`errTest` isn't defined in this file, but I can move it to the file it's defined in, sure! Done.